wiki/People/Beatmap_Nomination_Group/en.md

```diff
@@ -9,33 +9,33 @@
 [Ym]: /wiki/shared/Insane-m.png "Yes"
 
 Beatmap Nomination Group
-=========================
+========================
 
-The **Beatmap Nomination Group**, more commonly referred to as BN, is a group in charge of [beatmap nomination](/wiki/Beatmaps).
+The **Beatmap Nomination Group**, more commonly referred to as the BNG, is a group in charge of [beatmap nomination](/wiki/Submissions/Beatmap_Ranking_Procedure). Individual members are commonly referred to as BNs, or Beatmap Nominators.
```

Comment:
reverse the `BNs` and `Beatmap Nominators`
`Individual members are commonly referred to as Beatmap Nominators, or BNs.`